landoscript/src/landoscript/script.py

```diff
@@ -0,0 +1,73 @@
+import logging
+import os.path
+
+import aiohttp
+import scriptworker.client
+
+from landoscript import lando
+from landoscript.actions import version_bump
+from landoscript.github import GithubClient
+from scriptworker_client.github import extract_github_repo_owner_and_name
+
+log = logging.getLogger(__name__)
+
+
+def get_default_config(base_dir="") -> dict:
+    base_dir = base_dir or os.path.dirname(os.getcwd())
+    default_config = {
+        "work_dir": os.path.join(base_dir, "work_dir"),
+        "artifact_dir": os.path.join(base_dir, "artifact_dir"),
+        "schema_file": os.path.join(os.path.dirname(__file__), "data", "landoscript_task_schema.json"),
+    }
+    return default_config
+
+
+async def async_main(context):
+    config = context.config
+    payload = context.task["payload"]
+    artifact_dir = config["artifact_dir"]
+    public_artifact_dir = os.path.join(artifact_dir, "public", "build")
+    owner, repo = extract_github_repo_owner_and_name(payload["source_repo"])
+
+    os.makedirs(public_artifact_dir)
+
+    lando_actions = []
+    async with GithubClient(context.config["github_config"], owner, repo) as gh_client:
+        for action in payload["actions"]:
```

Comment:
Same here, I assume we'll want to protect actions via scopes too.

E.g, maybe we want to allow generating the changelog, but not bumping the version (just thinking of possible future use cases)